test/eth.accounts.encrypt-decrypt.js

```diff
@@ -18,98 +18,100 @@ var uuid = Buffer.from('ff31ddc3e2791ac53f3f19b125c18fff', 'hex');
 var pw = 'test';
 
 // tests from https://github.com/Gustav-Simonsson/go-ethereum/blob/7cc6b801e0967e5ebfa26b9f670675acea6e3a20/accounts/testdata/v3_test_vector.json
-var staticTests = [{
-    "json": {
-        "crypto" : {
-            "cipher" : "aes-128-ctr",
-            "cipherparams" : {
-                "iv" : "83dbcc02d8ccb40e466191a123791e0e"
-            },
-            "ciphertext" : "d172bf743a674da9cdad04534d56926ef8358534d458fffccd4e6ad2fbde479c",
-            "kdf" : "scrypt",
-            "kdfparams" : {
-                "dklen" : 32,
-                "n" : 262144,
-                "r" : 1,
-                "p" : 8,
-                "salt" : "ab0c7876052600dd703518d6fc3fe8984592145b591fc8fb5c6d43190334ba19"
+var staticTests = [
+    {
```

Comment:
I've formatted the JSON array correctly to increase the readability. Hopefully, this isn't out of the scope of this PR.